### DISCUSSION TITLE
Unable to join Slack channel

### DISCUSSION THREAD

**Discussion by garysassano** [skills/secure-code-game#67] [General]
I'm unable to join the GitHub Security Lab workspace.

![image](https://github.com/skills/secure-code-game/assets/10464497/539ebd25-d18c-4812-86f0-80c84ac433e2)

**Comment by jkcso** [comment]
Hey! Try this one please: https://ghsecuritylab.slack.com/signup#/domain-signup

**Reply by garysassano** [reply]
Hi Joseph, I still get the same error using that link you provided.

**Reply by jkcso** [reply]
Here you go: https://join.slack.com/t/ghsecuritylab/shared_invite/zt-2afhxkndc-3ZGRV1i2UMvCZ0DtXgqaQw
